examples/particles/js/editor/controls.js

```diff
@@ -46,7 +45,6 @@
             emitter.name = "emitter" + me.utils.createGUID();
             emitter.z = 10;
```

Comment:
yes so that's what i was saying/asking, based on thsi comment :
https://github.com/melonjs/melonJS/pull/731/files#r37757771